packages/effects-core/src/components/shape-component.ts

```
@@ -552,11 +548,16 @@ void main() {
 
-    //@ts-expect-error // TODO 新版蒙版上线后重构
-    material.stencilRef = data.renderer.mask !== undefined ? [data.renderer.mask, data.renderer.mask] : undefined;
-    //@ts-expect-error // TODO 新版蒙版上线后重构
-    setMaskMode(material, data.renderer.maskMode);
+    if (maskProps && maskProps.ref) {
+      maskProps.ref = this.engine.findObject((maskProps.ref as unknown as spec.DataPath));
+    }
+    const maskMode = this.maskManager.getMaskMode(data as MaskProps);
+    const maskRef = this.maskManager.getRefValue();
+
+    this.material.stencilRef = maskRef !== undefined ? [maskRef, maskRef] : undefined;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)